rhelai-validation/tasks/deploy_vm.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Git checkout the rhoso-rhelai roles
+  ansible.builtin.git:
+    repo: "{{ deploy_rhelai_repo }}"
+    dest: "{{ lookup('env', 'HOME') }}/rhoso-rhelai"
```

Comment:
Not yet sure if this will be the best location in the test-container or not

Reply:
In the test container, this will evaluate to: `/var/lib/ansible/rhoso-rhelai` which should be ok.

Reply:
This git checkout is going to go away (re-using this code turns out to be problematic in the test-operator container), but good to know that $HOME is an okay place for some temporary things in the container.